dxr/vcs.py

```diff
@@ -113,8 +114,9 @@ def find_previous_revisions(self, client):
         """
         last_change = {}
         for line in client.rawcommand(['previous-revisions']).splitlines():
-            path, node = line.split(':')
-            last_change[path] = node
+            if ':' in line:
+                path, node = line.split(':', 1)
+                last_change[path] = node
```

Comment:
Actually, we could probably avoid this whole thing by having previous_revisions print repr(filename) and then using https://docs.python.org/2/library/ast.html#ast.literal_eval to read it back in.